src/context.rs

```diff
@@ -1811,6 +1811,7 @@ pub struct ContextWriter {
   fc_map: Option<FieldMap> // For debugging purposes
 }
 
+#[allow(clippy::let_and_return)]
```

Comment:
Would it make more sense to move this down one line so it applies to only the `new` method, instead of the whole `impl ContextWriter`?

Reply:
Yes, you're right. Good catch. Fixed.